File: languages/django/law.py
class Law(AbstractElementModel):

    # Code
    decree = models.TextField(blank=True, null=True)
    date = models.PositiveIntegerField(blank=True, null=True)
    purpose = models.TextField(blank=True, null=True)
    author = models.ForeignKey("Institution", on_delete=models.SET_NULL, blank=True, null=True, related_name="law_author")

    # Compulsion
    jurisdictions = models.ManyToManyField("Location", blank=True, related_name="law_jurisdictions")
    prohibitions = models.ManyToManyField("Construct", blank=True, related_name="law_prohibitions")
    penalties = models.ManyToManyField("Construct", blank=True, related_name="law_penalties")
    adjudicators = models.ManyToManyField("Title", blank=True, related_name="law_adjudicators")
    enforcers = models.ManyToManyField("Title", blank=True, related_name="law_enforcers")

    def __str__(self):
        return self.name

File: conversions/csharp_unity_json_properties/Law.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Utils;

[System.Serializable]
public class Law : Element
{
    [JsonProperty("decree"), TextAttribute("")]
    public string decree;
    [JsonProperty("date"), Integer(0)]
    public int date;
    [JsonProperty("purpose"), TextAttribute("")]
    public string purpose;
    [JsonProperty("author"), ReferenceAttribute(typeof(Institution))]
    public string author;
    [JsonProperty("jurisdictions"), ReferenceAttribute(typeof(Location), true)]
    public string jurisdictions;
    [JsonProperty("prohibitions"), ReferenceAttribute(typeof(Construct), true)]
    public string prohibitions;
    [JsonProperty("penalties"), ReferenceAttribute(typeof(Construct), true)]
    public string penalties;
    [JsonProperty("adjudicators"), ReferenceAttribute(typeof(Title), true)]
    public string adjudicators;
    [JsonProperty("enforcers"), ReferenceAttribute(typeof(Title), true)]
    public string enforcers;
}
